test/execute.js

```diff
@@ -529,13 +529,60 @@ describe('execute', () => {
         headers: {
           head: 'justTheHead',
         },
-        body: {
+        body: JSON.stringify({
           json: 'rulez'
-        }
+        })
       })
     })
   })
 
+  it('should use the whole parameters object as the parameter value if parameter is in the body', function () {
```

Comment:
@webron this is about how the spec works. I'm not sure if what is proposed is valid. Please check.